e2e/setup/helpers.ts

```diff
@@ -135,14 +150,30 @@ export const buildFrontend = (
       NEXT_PUBLIC_BACKEND_HOSTNAME: config.backendHostname,
       NEXT_PUBLIC_OPEN_ID_CONNECT_MICROSOFT_SERVER: config.oidcUrl,
       NEXT_PUBLIC_OPEN_ID_CONNECT_MICROSOFT_CLIENT_ID: config.oidcClientId,
-      NEXT_PUBLIC_SCRATCH_APP_HOSTNAME: "http://localhost:3101/scratch",
+      NEXT_PUBLIC_SCRATCH_APP_HOSTNAME: "/scratch",
     },
     cwd: getFrontendPath(),
     shell: true,
     stdio: ["ignore", stdout, stderr],
   });
 };
 
+export const buildApp = (
+  app: CrtApp,
+  stdout: "pipe" | "ignore",
+  stderr: "pipe" | "ignore",
```

Comment:
since it seems we're using "pipe" everywhere, can we set it as default value? :-)

Reply:
I guess now we are so yes :)